spec/views/sign_up/recovery_codes/show.html.slim_spec.rb

```diff
@@ -43,4 +21,30 @@
 
     expect(rendered).to have_css('.step-3.active')
   end
+
+  context 'when render has no dependencies' do
```

Comment:
I guess it wouldn't, but it allowed me to group all the spec under it and use a before do rather than repeating the render call a bunch of times. I can just remove the block and explicitly render in each test.